front/src/components/Drawer.vue

```diff
@@ -63,6 +63,9 @@ export default {
     computed: {
       user() {
         return this.$store.getters.user;
+      },
+      userPicture() {
+        return this.user.photoURL || `https://eu.ui-avatars.com/api/?name=${this.user.email}`
```

Comment:
we risk to have redundant calls to ui-avatars api.
Is there a limit ? Maybe we can generate an avatar on our own ?